Form/DataTransformer/ModelsToArrayTransformer.php

```diff
@@ -42,11 +43,11 @@ class ModelsToArrayTransformer implements DataTransformerInterface
     /**
      * ModelsToArrayTransformer constructor.
      *
-     * @param ModelChoiceList       $choiceList
+     * @param ModelChoiceList|LazyChoiceList $choiceList
      * @param ModelManagerInterface $modelManager
      * @param                       $class
      */
-    public function __construct(ModelChoiceList $choiceList, ModelManagerInterface $modelManager, $class)
+    public function __construct($choiceList, ModelManagerInterface $modelManager, $class)
     {
         $this->choiceList   = $choiceList;
```

Comment:
Please add checks using `instanceof` that `$choiceList` is `ModelChoiceList` or `LazyChoiceList `but not an arbitrary object